packages/lms/src/lib/utils/fuseSearch/fuseSetup.ts

```diff
@@ -0,0 +1,63 @@
+import Fuse from 'fuse.js';
+import { parseMarkdownSearch } from '$lib/utils/parseMarkdownSearch';
+import { removeStopWords } from './removeStopWords';
+import { flattenData, type FuseItem } from './flattenData';
+
+const options = {
+	includeScore: true,
+	// threshold 0 is more strict, 1 is more loose
+	threshold: 0.3,
+	limit: Infinity,
```

Comment:
![neat](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExMmI4MzcwZjBiZmQ2ZDgxYjA0YjZjYWQ3NTU0NzczOTZiMTE3YzhmZSZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/8vtm3YCdxtUvjTn0U3/giphy.gif)

Never actually seen `Infinity` in use in the wild.

Reply:
- insert to infinity and beyond gif -

Reply:
It said I could use it, so why not, I guess I could tone it down a little.